### PR TITLE
Missed globalizing some warnings

### PR DESCRIPTION
**Pull request by 0candy**
* Missed globalizing a few warnings.
* Update to latest version of strong-globalize
* Fixed rootdir to use autonomousMsgLoading `all`

### CODE REVIEW

**Comment by 0candy** [comment]
@jannyHou PTAL. Thanks

File: app/index.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 var path = require('path');
 var SG = require('strong-globalize');
-SG.SetRootDir(path.join(__dirname, '..'));
+SG.SetRootDir(path.join(__dirname, '..'), {autonomousMsgLoading: 'all'});
```
Comment (jannyHou) [inline app/index.js]: I might miss something here, when do we need to set `autonomousMsgLoading` as `all`?

**Comment by jannyHou** [comment]
@0candy Only one question about the config var `autonomousMsgLoading`, others LGTM :+1:

File: app/index.js (hunk 1 above)
Comment (0candy) [inline app/index.js]: We need to add it to the entry point modules.

Comment (jannyHou) [inline app/index.js]: Do we need to update all repos to include it in entry point modules(like index.js in root folder), or it's only required in generator-lb?

Comment (0candy) [inline app/index.js]: We don't need to update all repos. Just the top modules. I will probably add it to `loopback` as well.

Comment (Amir-61) [inline app/index.js]: > Just the top modules. I will probably add it to loopback as well.
I'm just curious; is this needed for `loopback-datasource-juggler`, which is one of the main modules.

Comment (0candy) [inline app/index.js]: @Amir-61  Is it an entry point to a loopback app? Or will loopback and generator-loopback be accessed first?

Comment (Amir-61) [inline app/index.js]: >Is it an entry point to a loopback app?

No I don't think if `loopback-datasource-juggler` is the entry point.

I was just thinking of `loopback-datasource-juggler` based on this comment:
>We don't need to update all repos. Just the top modules. I will probably add it to loopback as well.

So if it is just for entry point modules, then I don't think if we needed it for `loopback-datasource-juggler` and we are good.

Thanks!

**Pull request merged**